src/components/main-panel/main-panel.tsx

```diff
@@ -137,8 +140,25 @@ export const MainPanel = () => {
         );
     }
 
+    const renderToast = () => {
+        if (toast && toast.length > 0) {
+            return toast.map((value) => {
+                return (
+                    <DisplayToast
+                        key={value.id}
+                        toast={value}
+                        toastFunction={value?.toastFunction}
+                    />
+                );
+            });
+        } else {
+            return null;
+        }
+    };
+
     return (
         <Wrapper $mode={renderMode}>
+            <ToastWrapper>{renderToast()}</ToastWrapper>
```

Comment:
- create a separate component to handle the rendering of all the toasts, this includes get the toasts from `useDisplay` and iterating through the array to render it
- move the toast messages from `MainPanel` to `FormBuilder` as the toast messages may not necessary be tied to the main panel

Reply:
changes made in the last push